gemsupervisor/src/common/GEMSupervisor.cc

```diff
@@ -270,18 +285,23 @@ void gem::supervisor::GEMSupervisor::initializeAction()
     if (m_useLocalDBInstance)
       p_gemDBHelper->connect(m_dbName.toString());
 
+    // do this only when RCMS is not present
     // for (auto i = v_supervisedApps.begin(); i != v_supervisedApps.end(); ++i) {
     auto initorder = getInitializationOrder();
     for (auto i = initorder.begin(); i != initorder.end(); ++i) {
-      for (auto j = i->begin(); j != i->end(); ++j) {
-        if (((*j)->getClassName()).rfind("tcds::") != std::string::npos) {
-          INFO("GEMSupervisor::initializeAction Halting " << (*j)->getClassName()
-               << " in case it is not in 'Halted'");
-          // need to ensure leases are properly respected
-          gem::utils::soap::GEMSOAPToolBox::sendCommand("Halt", p_appContext, p_appDescriptor, *j);
-        } else {
-          INFO("GEMSupervisor::initializeAction Initializing " << (*j)->getClassName());
-          gem::utils::soap::GEMSOAPToolBox::sendCommand("Initialize", p_appContext, p_appDescriptor, *j);
+      // if (!m_gemRCMSNotifier.getFoundRcmsStateListenerParameter()) {
+      if (true) {
```

Comment:
???

Reply:
It may at some point be necessary/useful to check if RCMS is controlling the supervisor, but this turned out to not be mandatory to a particular problem we were addressing in the `rcms` <--> `xdaq` cross communication, so I switched it to this so that everything would still "work", but that I could easily get back and properly implement the feature that I was trying to flesh out